linkerd/http-classify/src/layer.rs

```diff
@@ -0,0 +1,89 @@
+use super::{CanClassify, Classify};
+use futures::{try_ready, Future, Poll};
+
+#[derive(Debug, Clone)]
+pub struct Layer(());
+
+#[derive(Clone, Debug)]
+pub struct Make<N> {
```

Comment:
Nit/TIOLI: Elsewhere, it seems like we've been changing the naming for `Service`s/`MakeService`s to include the service name. While we're here, should this be changed to `MakeHttpClassify`? (and the `Service` struct `HttpClassify`?)